text/symbols.talon

```diff
@@ -14,41 +14,17 @@ dub arrow: "=>"
 new line: "\\n"
 carriage return: "\\r"
 line feed: "\\r\\n"
-empty dubstring:
-    '""'
-    key(left)
-empty escaped (dubstring|dub quotes):
-    '\\"\\"'
-    key(left)
-    key(left)
-empty string:
-    "''"
-    key(left)
-empty escaped string:
-    "\\'\\'"
-    key(left)
-    key(left)
-(inside parens | args):
-	insert("()")
-	key(left)
-inside (squares | square brackets | list):
-	insert("[]")
-	key(left)
-inside (bracket | braces):
-	insert("{}")
-	key(left)
-inside percent:
-	insert("%%")
-	key(left)
-inside (quotes | string):
-	insert("''")
-	key(left)
-inside (double quotes | dubquotes):
-    insert('""')
-	key(left)
-inside (graves | back ticks):
-	insert("``")
-	key(left)
+empty dubstring: user.insert_between('"', '"')
+empty escaped (dubstring|dub quotes): user.insert_between('\\"', '\\"')
+empty string: user.insert_between("'", "'")
+empty escaped string: user.insert_between("\\'", "\\'")
```

Comment:
Huh I personally use "empty" to mean don't leave cursor inside.  But I guess you're not changing anything here 🤷

Reply:
Yeah, I do the same thing, but this appears to be the default.